src/config.js

```diff
@@ -579,6 +580,9 @@ export default class Config {
 
       await fs.writeFilePreservingEol(loc, JSON.stringify(object, null, indent || constants.DEFAULT_INDENT) + '\n');
     }
+
+    // clear the cache, otherwise the new manifests can't be loaded through this config
+    this.cache = map();
```

Comment:
Not anymore, I'll remove it (basically, the old behavior was to execute a script. However, in order to get this script, I had to clear the package.json cache, since it had stored before during the then-local `add`).